src/styleguide/template.html

```diff
@@ -87,5 +87,36 @@
     </div></div>
 
     <script src="assets/js/app.js"></script>
+<script type="text/javascript">
+$(function() {
+    var colorsNames = [];
+    $("[data-color-block]").each(function() {
+        var code = rgbToHex($(this).css("background-color"));
+        var name = $(this).data("color-block");
+        colorsNames.push({
+            "name": name,
+            "code": code
+        });
```

Comment:
I took your previous recommendation literally and made the key "name, code" will fix and repush